### ISSUE TITLE
Is this functional on OSX?

### ISSUE THREAD

**Issue by pag-crypto** [rien/reStream#6]
I have a 2019 Macbook Air running Catalina version 10.15.2. I used Homebrew to install lz4 and ffmpeg and tried to run reStream using the shell script. It prompts me to enter the reMarkable's password three times, then starts ffplay. However, no window pops up and ffplay does not appear to receive any input from the tablet when I'm drawing on it. I've attached a screenshot. (An ffplay task is shown in the taskbar, but it's not drawing anything.)
![Screen Shot 2020-03-27 at 5 59 28 PM](https://user-images.githubusercontent.com/25438281/77803928-b980a100-7054-11ea-8770-9900046ade29.png)

**Comment by pag-crypto** [comment]
Actually, I just got it to work (to some extent) - the problem has to do with me needing to re-type the reMarkable password every time ssh is called, so I'm going to try to fix that first. Feel free to close; I might keep posting notes to help the next person who has this issue.

**Comment by pag-crypto** [comment]
I figured out what the problem is - without having a working ssh key in authorized_keys on the remarkable, each ssh prompt requires a password input which eventually breaks it. I solved it by generating an ssh key pair just for tablet screen sharing, using ssh-copy-id to load it into the tablet, and modifying the ssh calls in the script to take the -i flag. 

Bottom line: if you need to type the reMarkable's password to login via ssh, this script will ***not work***.

**Comment by rien** [comment]
You could [create an ssh
key](https://help.github.com/en/enterprise/2.15/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) and copy it to the reMarkable using `ssh-copy-id` so you don't have to enter the password all the time.

**Comment by pag-crypto** [comment]
(By the way - thank you so much for making this, I think it's extremely cool and makes my tablet much more useful for research meetings!)

**Comment by acrogenesis** [comment]
@pag-crypto how were you able to solve the black screen?
I only see this:
<img width="2048" alt="CleanShot 2020-08-10 at 13 42 25@2x" src="https://user-images.githubusercontent.com/1312687/89818442-5ed17880-db0f-11ea-9886-8664c81fa559.png">

**Comment by pag-crypto** [comment]
@acrogenesis See my post from March 27th - my problem was that the script doesn't work on OSX if SSHing into the reMarkable requires typing a password. I fixed it by setting up an SSH key and modifying the reStream.sh script to pass the key as a parameter (via -i) to all the calls to SSH.

In case that's ambiguous or confusing, I'm attaching [my modified reStream.sh](https://github.com/rien/reStream/files/5053019/reStream.copy.txt) for you to look at. I called my SSH key 'id_rsa_remarkable'.
